cla_backend/apps/reports/sql/MIExtractByOutcomeTellUsAboutYourProblem.sql

```diff
@@ -28,7 +27,6 @@ SELECT
   ,COALESCE(log_changed_category.diagnosis_category, category.code) as "Diagnosis Category"
   ,category.code as "Legalaid Category Code"
   ,category.name as "Legalaid Category Name"
-  ,category.description as "Legalaid Category Description"
```

Comment:
This one is on the ticket, any reason it was removed?

Reply:
The field returns a description of the legal category selected, the ticket states Operator’s most recent categorisation of client’s legal issue; this is covered in Legalaid Category Name, the full description is not required